README.md

```diff
@@ -1 +1,21 @@
 # LFP Appeals Processor API
+
+## How to run
+### Run as a Spring Boot application
+`mvn clean install spring-boot:run`
+
+## Useful Endpoints
+### Health
+http://localhost:9000/actuator/health
+
+### API Documentation (Swagger)
+http://localhost:9000/swagger-ui.html
+
+## Technologies
+- [OpenJDK 11](https://jdk.java.net/archive/)
+- Maven 3.x.x
+- Spring Boot 2.2.x
+- Swagger OpenAPI 3.0
+
+
```

Comment:
I am curious your feeling here cause I think excessive blank lines are unnecessary. Although they does make much harm it makes code look more sloppy IMO. I am curious your and team's opinion here.

Reply:
good spot - I didn't realise I had added the blank lines! I like the idea of line spaces between titles and text, but they're not needed a the end of the file